benchmarks/mongo-kafka-snapshot/conduit/compose.override.yml

```diff
@@ -0,0 +1,7 @@
+services:
+  benchi-conduit:
+    environment:
+      CONDUIT_PREVIEW_PIPELINE_ARCH_V2: true
```

Comment:
We already have this in shared/conduit/compose.yml, right?

Reply:
This doesn't work anyway (https://github.com/ConduitIO/conduit/issues/2206) 😢 It's a leftover, will remove!